platforms/nuttx/CMakeLists.txt

```diff
@@ -297,6 +297,29 @@ else()
 
 endif()
 
+# create .hex if configured in NuttX
+if (CONFIG_INTELHEX_BINARY)
+
+	string(REPLACE ".elf" ".hex" hex_package ${PX4_BINARY_DIR}/${FW_NAME})
+
+	add_custom_command(
+		OUTPUT ${hex_package}
+		COMMAND
+			arm-none-eabi-objcopy -O ihex
```

Comment:
Dev call @dagar mentioned this command should be taken from the CMake detection.

Reply:
Thanks using `${CMAKE_OBJCOPY}` now instead.